pinot-segment-local/src/main/java/org/apache/pinot/segment/local/realtime/impl/invertedindex/RealtimeLuceneTextIndex.java

```diff
@@ -181,6 +181,17 @@ private MutableRoaringBitmap getPinotDocIds(IndexSearcher indexSearcher, Mutable
     return actualDocIDs;
   }
 
+  @Override
+  public void commit() {
+    try {
+      _indexCreator.getIndexWriter().commit();
+    } catch (Exception e) {
+      LOGGER.error("Failed to commit the realtime lucene text index for column {}, exception {}", _column,
+          e.getMessage());
```

Comment:
why not just log the error e?

Reply:
didn't quite follow this, can you elaborate?

Reply:
I meant LOGGER.err("Failed.. .", _column, e);